src/highlighted-card.tsx

```diff
@@ -104,50 +67,56 @@ const Content: React.FC<Props> = (props) => {
     const {title, description, imageUrl, imageFit} = props;
     const isInverseOutside = useIsInverseVariant();
     const isInverse = props.isInverse ?? isInverseOutside;
-    const classes = useStyles({isInverse, hasImage: !!imageUrl, width: props.width});
-    const theme = useTheme();
     const isDismissable = useIsDismissable();
 
     const content = (
-        <Boxed isInverse={isInverse} className={classes.container} dataAttributes={props.dataAttributes}>
-            <div
-                // don't create another region when the Content is inside a Dismissable wrapper
-                role={!isDismissable ? 'region' : undefined}
-                className={classes.textContainer}
-                // aria-label is already in Dismisable wrapper
-                aria-label={!isDismissable ? props['aria-label'] : undefined}
-            >
-                <Text4 as="h1" regular truncate={props.titleLinesMax}>
-                    {title}
-                </Text4>
-                <Box paddingTop={8}>
-                    <Text2
-                        regular
-                        color={theme.colors.textSecondary}
-                        truncate={props.descriptionLinesMax}
-                        as="p"
-                    >
-                        {description}
-                    </Text2>
-                </Box>
-                {props.button && (
-                    <>
-                        <div style={{minHeight: 16, flexGrow: 1}} />
-                        {props.button}
-                    </>
-                )}
-            </div>
-            {imageUrl && (
+        <div
+            style={{
+                ...assignInlineVars({[styles.vars.width]: props.width ? `${props.width}px` : '100%'}),
+                display: 'flex',
+                width: '100%',
+            }}
```

Comment:
new wrapper needed to set css var, any other idea?

Reply:
you can set the var to the nodes that use it instead